src/Promitor.Integrations.Sinks.Prometheus/Collectors/AzureScrapingSystemMetricsPublisher.cs

```diff
@@ -48,5 +48,14 @@ public async Task WriteGaugeMeasurementAsync(string name, string description, do
         {
             await _systemMetricsPublisher.WriteGaugeMeasurementAsync(name, description, value, labels, includeTimestamp);
         }
+        
+        public async Task WriteHistogramMeasurementAsync(string name, string description, double value, Dictionary<string, string> labels)
```

Comment:
Don't need async here

Reply:
Hm are you sure? I thought `async` is needed when you await in the body?